arch/arm64/boot/dts/qcom/msm8953-xiaomi-ysl.dts

```diff
@@ -127,6 +126,15 @@
 	compatible = "xiaomi,ysl-panel";
 };
 
+&pm8953_l6 {
+	regulator-always-on;
+};
+
+&pm8953_l10 {
+	regulator-min-microvolt = <2800000>;
+	regulator-max-microvolt = <3000000>;
+};
```

Comment:
Can you add a comment to both l6 and l10 why they're different here or something. Especially the always-on should have a reason

Reply:
I don't really know a good comment for this, i just saw downstream uses different voltage values for l10 and there is no always-on definitions in the common.
I have checked other devices and tissot regulators also differs from the common in downstream.

Reply:
No schematics available? Maybe git history downstream might help? Assuming Xiaomi doesn't just publish an archive without git history.

Reply:
I have checked L10 related to sensors and default value should be 2.800 v but i don't find any info about max why should be 3v. L6 also used by the sensors.
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/commit/09f6450330b57b8df8b4472917c2d71adc0386d4#diff-1d92316904c11eef1f77aa098cc2b3c418bf8ff1e1fcb7f9fef834f3b8ef0b2eL219-R228

Reply:
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/blob/09f6450330b57b8df8b4472917c2d71adc0386d4/arch/arm/boot/dts/qcom/msm8953-mtp.dtsi#L149C1-L151C3 it is setting back the max voltage to 2.85v
:(

Reply:
I have build and decompiled the downstream devicetrees
Android P:
```
 regulator-l10 {
          compatible = "qcom,rpm-smd-regulator";
          regulator-name = "pm8953_l10";
          qcom,set = <0x03>;
          status = "okay";
          regulator-min-microvolt = <0x2ab980>;
          regulator-max-microvolt = <0x2b7cd0>;
          qcom,init-voltage = <0x2ab980>;
          regulator-always-on;
          linux,phandle = <0xd5>;
          phandle = <0xd5>;
  };
```
Android O:
```
regulator-l10 {
        compatible = "qcom,rpm-smd-regulator";
        regulator-name = "pm8953_l10";
        qcom,set = <0x03>;
        status = "okay";
        regulator-min-microvolt = <0x2ab980>;
        regulator-max-microvolt = <0x2b7cd0>;
        qcom,init-voltage = <0x2ab980>;
        linux,phandle = <0xd5>;
        phandle = <0xd5>;
};
```